src/pages/gen2/build-a-backend/auth/override-cognito/index.mdx

```diff
@@ -21,12 +21,65 @@ import { defineAuth } from '@aws-amplify/backend';
 export const auth = defineAuth({
   loginWith: {
     email: true
-  }
+  },
 });
 ```
 
 You can customize your authentication resource to enable it to behave exactly as needed for your use cases by modifying it directly using [AWS Cloud Development Kit (CDK)](https://aws.amazon.com/cdk/)
 
+## Customize Cognito UserPool email verification settings
+
+You can change the verification email by overriding the `verificationEmailStyle` and the `verificationEmailBody`.
+
+```ts title="amplify/auth/resource.ts"
+import { defineAuth } from "@aws-amplify/backend";
+
+export const auth = defineAuth({
+  loginWith: {
+    email: {
+      verificationEmailStyle: "CODE",
+      verificationEmailBody: (code: string) =>
+        `Welcome! Your verification code is ${code}.`,
+      verificationEmailSubject: "Welcome! Here is your verification code",
+    },
+  },
+});
+```
+
+## Override Cognito UserPool password policies
+
+You can override the password policy by using the L1 `cfnUserPool` construct and adding a `addPropertyOverride`.
+
+```ts title="amplify/backend.ts"
+import { Backend } from '@aws-amplify/backend';
+import { auth } from './auth/resource';
+
+const backend = new Backend({
+  auth,
+});
+
+// L1 construct cfnUserPool
+const { cfnUserPool } = backend.resources.auth.resources.cfnResources;
+// override userpool password policies
+cfnUserPool.addPropertyOverride(
+  "Policies",
+  {
+    PasswordPolicy: {
+      MinimumLength: 10,
+      RequireLowercase: true,
+      RequireNumbers: true,
+      RequireSymbols: true,
+      RequireUppercase: true,
+      TemporaryPasswordValidityDays: 20,
+    },
+  }
+);
+```
```

Comment:
```suggestion
```ts title="amplify/backend.ts"
// amplify/backend.ts
import { defineBackend } from '@aws-amplify/backend';
import { auth } from './auth/resource.js';

const backend = defineBackend({
  auth,
});

// extract L1 CfnUserPool resources
const { cfnUserPool } = backend.resources.auth.resources.cfnResources;
// use CDK's `addPropertyOverride` to modify properties directly
cfnUserPool.addPropertyOverride(
  "Policies",
  {
    PasswordPolicy: {
      MinimumLength: 10,
      RequireLowercase: true,
      RequireNumbers: true,
      RequireSymbols: true,
      RequireUppercase: true,
      TemporaryPasswordValidityDays: 20,
    },
  }
);
```
```

Reply:
few adjustments on this one:
- Backend -> defineBackend
- copy/pasted comments from the other example
- add a comment with the filename at the top of the example

Reply:
Will do